### ISSUE TITLE
z-wave connection

### ISSUE THREAD

**Issue by Reemster** [athombv/homey#2823]
After the latest beta update (rc11) my zwave is not stable anymore. Or it's not working at all. Or the dimmer (or other zwave devices) working with delay. 

maybe you can see whats happend : log 2E7C7A19E3

**Comment by peidy28** [comment]
yup, same problem here, connection is lost at random.  delays are terrible . random Tamper alerts , motion detection keeps blinking, lights dont respond ,

**Comment by AdyRock** [comment]
Same issue for me too. Z-Wave now unreliable with 1.5.13-rc11. At first I thought it was just reduced range as things seemed better when homey was just a few metres away from the device. However even then it is not as solid as it was. If I put homey in it's original position, the connection rarely works. I have also found my Qubino  shutter DC is now using more power than the solar panel can produce so the batteries are only last a few days and then I have to put them in a charger. When I was testing that a few months back the battery pack lasted for over two weeks of testing without the solar panel. So makes me wonder if it is constantly retrying connections and not sleeping.
Not sure if it is related or another issue but my Lightwave RF wireless switch only gets detected about 1 in 10 presses now as well. Again distance doesn't seem to be related as it is the same from a few cm or 10 m. I have checked with my RFXCom module and the signal is picked up every time even over 20m away so the switch seems to be fine.
So maybe the issue is with general radio rather than Z-Wave specifically.

**Comment by Tim81** [comment]
Done a few tests with moving homey around and it looks like the z-wave range of homey is a lot lower then previous firmwares.

**Comment by philsniff** [comment]
i also have these issues. from time to time there are noacks in the zwave network. never had this before rc.11. after some tries it works again or with some delay which was not before. so this issue is really there but random.

**Comment by cjvs** [comment]
Same here, since 1.5.13-rc.11 fibaro delays in lights switching on or off, generally only when done with a flow where multiple lights / actions are preformed (e.g. like switchin off all lights in a room). Also had a KaKu switch failing from time to time. Random so not possible to re-create.
